barretenberg/cpp/src/barretenberg/vm2/tracegen_helper.cpp

```diff
@@ -184,6 +204,7 @@ TraceContainer AvmTraceGenHelper::generate_trace(EventsContainer&& events)
         auto jobs_interactions = make_jobs<std::unique_ptr<InteractionBuilderInterface>>(
             std::make_unique<LookupIntoBitwise<lookup_dummy_precomputed_lookup_settings>>(),
             std::make_unique<LookupIntoDynamicTableGeneric<lookup_dummy_dynamic_lookup_settings>>(),
+            std::make_unique<LookupIntoDynamicTableGeneric<lookup_pos2_perm_lookup_settings>>(),
```

Comment:
I think we don't expect to deduplicate for poseidon2 and poseidon2perm right? In that case you can use `LookupIntoDynamicTableSequential`